docs/getstarted/educators-and-students.md

```diff
@@ -0,0 +1,152 @@
+
+The following short clip shows a student using CodeTour to start working on their app:
+
+![CodeTour in a Space Jam lesson](images/educators-and-students/codetour-space-jam.gif)
+
+Essentially, a "code tour" is a series of interactive steps, each of which are associated with a specific directory, or file/line, and include a description of the respective code. Tours can either be checked into a repo to share with source control, or exported to a "tour file", which enables anyone to replay the same tour, without having to clone any code!
+
+In the following video, an instructor is removing existing instructions embedded in the source code and replacing them with CodeTour steps. Students can then jump from step to step in the large game development project.
+
+![CodeTour in a sample assignment](images/educators-and-students/codetour-example-lesson.gif)
+
+#### Getting started with CodeTour
+
+To start using CodeTour, you need to download the CodeTour extension from the VS Code [Marketplace](https://marketplace.visualstudio.com/vscode).
+
+![CodeTour extension in Extension Marketplace](images/educators-and-students/codetour-extension-marketplace.png)
+
+To use CodeTour effectively in your classroom, we recommend following the guidelines in the [CodeTour repository](https://github.com/microsoft/codetour#getting-started).
+
+## Collaborate with Live Share
+
+### What is Live Share?
+
+[Live Share](https://learn.microsoft.com/visualstudio/liveshare) enables you to quickly collaborate with a friend, classmate, or professor on the same code without the need to sync code or to configure the same development tools, settings, or environment.
+
+When it comes to Live Share, seeing is believing. Check out this video to see Live Share in action:
+
+<iframe src="https://youtube.com/embed/A2ceblXTBBc?rel=0&amp;disablekb=0&amp;modestbranding=1&amp;showinfo=0" frameborder="0" allowfullscreen title="Collaborate with Live Share"></iframe>
+
+>**Note**: The video mentions the deprecated Live Share Extension Pack and Live Share Audio extension. To follow along with the video, you only need to install the [Live Share](https://marketplace.visualstudio.com/items?itemName=MS-vsliveshare.vsliveshare) extension.
+
+When you share a collaborative session, the person you're working with sees the context of the workspace in their editor. This means your classmate can read the code you shared without having to clone a repo or install any dependencies your code relies on. They can help you with your code in the Visual Studio Code environment that's familiar to them.
```

Comment:
As a user potentially newer to VS Code, I'm not sure I know what is included in / meant by `the context of the workspace in their editor`.

Reply:
I'll rephrase this section.

Reply:
I made some changes to this section. What do you think?

Reply:
Looks great!